dev-python/dj-search-url/dj-search-url-0.1.ebuild

```diff
@@ -1,9 +1,10 @@
 # Copyright 2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{8..10} )
+DISTUTILS_USE_PEP517=setuptools
```

Comment:
Need revbump for PEP517 conversions b/c they're risky and also might change installed files (image).

Reply:
whoops, I had a feeling that I missed one!